src/test/scala/chiselTests/CompatibilitySpec.scala

```diff
@@ -17,13 +18,13 @@ object CompatibilityCustomCompileOptions {
   }
 }
 
-class CompatibiltySpec extends ChiselFlatSpec with ScalaCheckDrivenPropertyChecks {
+class CompatibiltySpec extends ChiselFlatSpec with ScalaCheckDrivenPropertyChecks with Utils {
```

Comment:
One is forced to wonder what is being checked for compatibility here. Can we deprecate this and get rid of it at some point or does it perhaps just need a new name

Reply:
Agree. A lot of this is simply checking that classes, objects, and methods aren't removed.